climada_petals/hazard/test/test_tc_tracks_forecast.py

```diff
@@ -78,6 +78,27 @@ def test_equal_timestep(self):
         self.assertEqual(forecast.data[1].environmental_pressure.size, 13)
         self.assertEqual(forecast.data[1].time_step[2], 1.)
 
+    def test_hdf5_io(self):
+        """Test writting and reading hdf5 TCTracks instances"""
+        tc_track = TCForecast()
+        tc_track.fetch_ecmwf(files=TEST_BUFR_FILES)
```

Comment:
Are these test files already on the API? If not, it might be a good moment to do so and adapt the test accordingly... What do you think @emanuel-schmid ?

Reply:
These two files are about 6 KB in size. I suggest we leave them in the repository for now. If we want to move them to the API I would appreciate some guidance or help.